composer.json

```diff
@@ -34,18 +34,12 @@
 			"dealerdirect/phpcodesniffer-composer-installer": true
 		}
 	},
-	"repositories": [
-		{
-			"url": "https://github.com/buddypress/bp-coding-standards.git",
-			"type": "git"
-		}
-	],
 	"require": {
 		"composer/installers": "^1.10.0",
 		"php": ">=5.6.0"
 	},
 	"require-dev": {
-		"buddypress/bp-coding-standards": "dev-trunk",
```

Comment:
I'll be making some changes to the dev-trunk soon. So to avoid any issues here, I'm changing it to install from the stable version.